### PR TITLE
Simplify the demo of plugin ca cert.

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: _docs/tasks/security/plugin-ca-cert.md [hunk 1]
```diff
@@ -35,20 +35,14 @@ which should include the certificates of all the intermediate CAs between the wo
...
 
-   Download the example files:
-   ```bash
-   rm /tmp/ca-cert.pem /tmp/ca-key.pem /tmp/root-cert.pem /tmp/cert-chain.pem
-   wget -P /tmp https://raw.githubusercontent.com/istio/istio/master/security/samples/plugin_ca_certs/ca-cert.pem
-   wget -P /tmp https://raw.githubusercontent.com/istio/istio/master/security/samples/plugin_ca_certs/ca-key.pem
-   wget -P /tmp https://raw.githubusercontent.com/istio/istio/master/security/samples/plugin_ca_certs/root-cert.pem
-   wget -P /tmp https://raw.githubusercontent.com/istio/istio/master/security/samples/plugin_ca_certs/cert-chain.pem
-   ```
+These files are ready to use in install/kubernetes/ directory.
```
Comment (geeknoid) [inline _docs/tasks/security/plugin-ca-cert.md]: in **the** `install\kubernetes`

Comment (wattli) [reply _docs/tasks/security/plugin-ca-cert.md]: done, thanks

**Pull request merged**